bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/internal/DeviceManagerFactory.java

```diff
@@ -79,7 +79,13 @@ public void initialize() {
                 // Experimental - seems reuse does not work
             } catch (IllegalStateException e) {
                 // Exception caused by first call to the library
-                return DeviceManager.createInstance(false);
+                try {
+                    return DeviceManager.createInstance(false);
+                } catch (DBusException ex) {
+                    // we might be on a system without DBus, such as macOS or Windows
+                    logger.debug("Failed to initialize DeviceManager: {}", ex.getMessage());
+                    return null;
+                }
```

Comment:
Catching and logging the exception here isn't really necessary since this exception would get handled in the `whenComplete` later on. Also by not returning a `null` here you don't have to add `@Nullable ` on line 54 and all of the null checks wouldn't be required either.

Reply:
That's what I thought as well, but it wasn't the case. I had to add this catch additionally to not make exceptions popping up in the log.
Feel free to create an alternative PR with less changes and I can test it.

Reply:
> Feel free to create an alternative PR with less changes and I can test it.

I'll get to work on that.